osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -94,14 +95,17 @@ protected override void LoadComplete()
 
                 Combo.ValueChanged += OnComboChange;
 
-                this.FadeInFromZero(FLASHLIGHT_FADE_DURATION);
-
-                foreach (var breakPeriod in Breaks)
+                using (BeginAbsoluteSequence(0))
```

Comment:
Might want to check if this is how stable does it. It means the flashlight will nto be visible during lead-in time.

Reply:
I could leave this change for a separate PR, currently flashlight fades on master are incorrectly accounting for break time and this change attempts to fix that (see Disco Prince with flashlight as an example). For the entire intro sequence to match stable, it needs a little more work. On stable it does a bit of an expand and then contract, which lazer currently does not do. I'll probably open/edit the current issue with a list of things that flashlight needs to match stable.

The flashlight is visible during the lead-in time because a FadeInFromZero is applied immediately on LoadComplete above this change. There's a separate issue where it attempts to re-apply the flashlight when gameplay starts though. Probably even more reason to make this a different PR.

Reply:
I don't follow your response here. You say it was applied "immediately" but I cannot see where that was being done (before you made changes after my review). Please explain what you mean.

Reply:
I'm referring to the dim applied on https://github.com/ppy/osu/blob/b2b7143ed328a6d653ed3ce1fb3cdd34594e6f4c/osu.Game/Rulesets/Mods/ModFlashlight.cs#L98

Reply:
you mean the one that was moved after my comment..?

Reply:
yeah, as I mentioned in the above comment, that was incorrect.